### PR TITLE
Add lara  w3id

### PR DESCRIPTION
**Pull request by markdoerr**
Please add lara w3id

LARA (Laboratory Automation Robotic Assistent) is an open source Research Data Management Suite and 2nd generation Electronic Lab Notebook with the goal to automate most of the data and metadata entering process by providing a rich infrastructure for lab communication, orchestration, data/metadata storage, evaluation and exchange. It utilises Ontology based semantic web technology to make data findable via SPARAQL.

Thanks :)

### CODE REVIEW

File: lara/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+LARA
+=====
+
+LARA (Laboratory Automation Robotic Assistent) is an open source Research Data Management Suite and 2nd generation Electronic Lab Notebook with the goal to automate most of the data and metadata entering process by providing a rich infrastructure for lab communication, orchestration, data/metadata storage, evaluation and exchange. It utilises Ontology based semantic web technology to make data findable via SPARAQL.
```
Comment (TallTed) [inline lara/README.md:4]:
```suggestion
LARA (Laboratory Automation Robotic Assistent) is an open source Research Data Management Suite and 2nd generation Electronic Lab Notebook with a goal of automating most of the process of entering data and metadata by providing a rich infrastructure for lab communication, orchestration, data/metadata storage, evaluation, and exchange. It uses ontology-based semantic web technology to make data findable via SPARQL.
```

File: oso/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+OSO
+===
+
+The Open Science Ontologies - (OSO) are a set of lean, open and modular top-level and domain ontologies designed for interoperable scientific experimentation, fast reasoning and machine learning / AI applications in mind.
```
Comment (TallTed) [inline oso/README.md:4]:
```suggestion
The Open Science Ontologies (OSO) are a set of lean, open, and modular top-level and domain ontologies designed for interoperable scientific experimentation, fast reasoning, and machine-learning/AI applications. 
```

**Comment by dgarijo** [comment]
@markdoerr please review and commit the suggestions by @TallTed if you agree. Thanks!

**Comment by markdoerr** [comment]
Dear @TallTed and @dgarijo,
thanks for your native-speaker inputs :)
(and sorry for the late reply - I had to work on several submission deadlines)

**Pull request merged**